### PR TITLE
Fix #158: Specify which arrays must be unique

### LINKED ISSUE #158: Specify which arrays must have unique items

### PR DESCRIPTION
**Pull request by ghost**
Also:
- Move a couple of references to the "Normative" section.

@michaelcfanning

### CODE REVIEW

File: document/Bibliography.mdk [hunk 1]
```diff
@@ -1,8 +1,5 @@
 # Bibliography { - }
 
-ISO 8601:2004, _Data elements and interchange formats -- Information interchange -- Representation of dates and times_.
-Available from  &lt;<http://www.iso.org/iso/catalogue_detail?csnumber=40874>&gt;
-
```
Comment (ghost) [inline document/Bibliography.mdk:5]: Moved to "Normative"

File: document/Normative.mdk [hunk 2]
```diff
@@ -11,7 +11,15 @@ Available from &lt;<http://ecma-international.org/publications/files/ECMA-ST/ECM
 FIPS PUB 180-4, Secure Hash Standard (SHS).
 Available from &lt;<http://csrc.nist.gov/publications/fips/fips180-4/fips-180-4.pdf>&gt;
 
+ISO 8601:2004, _Data elements and interchange formats -- Information interchange -- Representation of dates and times_.
+Available from  &lt;<http://www.iso.org/iso/catalogue_detail?csnumber=40874>&gt;
+
+JSON Schema: core definitions and terminology \[viewed 2016-04-22\]. Available from &lt;<http://json-schema.org/latest/json-schema-core.html>&gt;
+
```
Comment (ghost) [inline document/Normative.mdk:18]: Added

File: document/Normative.mdk [hunk 3]
```diff
@@ -11,7 +11,15 @@ Available from &lt;<http://ecma-international.org/publications/files/ECMA-ST/ECM
 FIPS PUB 180-4, Secure Hash Standard (SHS).
 Available from &lt;<http://csrc.nist.gov/publications/fips/fips180-4/fips-180-4.pdf>&gt;
 
+ISO 8601:2004, _Data elements and interchange formats -- Information interchange -- Representation of dates and times_.
+Available from  &lt;<http://www.iso.org/iso/catalogue_detail?csnumber=40874>&gt;
+
```
Comment (ghost) [inline document/Normative.mdk:16]: Moved from Bibliography

File: document/Normative.mdk [hunk 4]
```diff
@@ -11,7 +11,15 @@ Available from &lt;<http://ecma-international.org/publications/files/ECMA-ST/ECM
...
+JSON Schema: core definitions and terminology \[viewed 2016-04-22\]. Available from &lt;<http://json-schema.org/latest/json-schema-core.html>&gt;
+
 RFC 2045, Multipurpose Internet Mail Extensions (MIME) Part One: Format of Internet Message Bodies.
 Available from &lt;<http://www.ietf.org/rfc/rfc2045.txt>&gt;
 
+RFC 3986, Uniform Resource Identifier (URI): Generic Syntax.
+Available from &lt;<https://tools.ietf.org/html/rfc3986>&gt;
+
```
Comment (ghost) [inline document/Normative.mdk:24]: Moved from Bibliography

**Comment by michaelcfanning** [comment]
+1

**Pull request merged**